processing/src/main/java/io/druid/segment/StorageAdapter.java

```diff
@@ -54,15 +54,16 @@
   @Nullable
   Comparable getMaxValue(String column);
   Capabilities getCapabilities();
+  boolean areDimensionValuesSorted();
```

Comment:
This class doesn't have enough javadoc, so let's make it better. Please add javadoc for this method.